### PR TITLE
Remove select_age module

### PR DESCRIPTION
**Pull request by bregman-arie**
select_age module contains one method that can be included
in resources.

In my opinion it should be contained in resources since
there is no real use for it as a seperate object. It only
selects resources (as resource module) but by age parameters.

In addition, it will allow us to move all "old" and "unused" logic
from janitor main loop to resources module where it should be.

### CODE REVIEW

**Comment by yazug** [comment]
resources is an additive collection object with no reset right now.  This refactor Breaks the age related cleanup behavior.  Prior behavior it would clean up one old instance per run, this branch attempts to cleanup up all of them.  

If you want to change behavior please make that a separate pull request, if you are refactoring please maintain all functionality in the refactor.

Will not merge as is.

File: shade_janitor/janitor.py [hunk 1]
```diff
@@ -76,9 +76,8 @@ def set_logging(debug_mode):
     cleanup = {}
 
     if args.old_instances:
-        age_resources = SelectAgeRelatedResources(cloud)
```
Comment (yazug) [inline shade_janitor/janitor.py]: I think you forgot to include cleanup of the import line above

**Comment by yazug** [comment]
Also I'd like to see unittest around the age selection functionality before it is moved.

**Comment by bregman-arie** [comment]
"Prior behavior it would clean up one old instance per run, this branch attempts to cleanup up all of them"

Why would you want it cleanup only one per run? I'm not sure if it's clear for users who use it. I would expect it to cleanup all old instances..

**Pull request closed without merging**